.github/workflows/windows.yml

```diff
@@ -59,8 +59,7 @@ jobs:
           call "C:\Program Files (x86)\Microsoft Visual Studio\${{ matrix.vs }}\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
           set YACC=win_bison
           echo on
-          nmake up
-          nmake extract-gems
+          nmake prepare-gems
```

Comment:
`up` does things other than `update-gems` too.
This change makes `fiddle` to be skipped.

Reply:
Right, sorry, I thought it was an alias to `update-gems`. Let me revert this.